### PR TITLE
fix: #1316 Update to Demo Site Search Widget

### PR DESCRIPTION
**Pull request by phmngocnghia**
# Pull request checklist

## Does this close any currently open issues?

fixes: #1316

**Please check if your PR fulfills the following requirements:**

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`yarn build`) was run locally and any changes were pushed
- [x] Lint (`yarn lint`) has passed locally and any fixes were made for failures
- [x] Test (`yarn test`) has passed locally and any fixes were made for failures

## Pull request type

**Please check the type of change your PR introduces:**

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?
- Advance search button mis-aligned, and book valuation theme
image
- Advance search button color is difference with For sale, For rent button
image

## What is the new behavior?
- Fix all stuff above
image

image

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/web-components/src/common/styles/theme.ts [hunk 1]
```diff
@@ -24,6 +24,8 @@ export interface InitializerTheme {
 }
 
 export interface ThemeClasses {
+  searchWidgetAdvanceSeachLabel: string
```
Comment (willmcvay) [inline packages/web-components/src/common/styles/theme.ts:27]: Can these keys be made more generic please so they can be re-used.

Suggest;

`featureButton` and `featureLabel` so can be re-used across components without binding their use semantically to the search widget.

File: packages/elements/src/styles/components/form.scss [hunk 2]
```diff
@@ -1,25 +1,25 @@
-
 @import '../base/colors.scss';
 @import '../base/fonts.scss';
 
-.form {
+.form2 {
```
Comment (willmcvay) [inline packages/elements/src/styles/components/form.scss:4]: Can we call this something more meaningful than form2 please?

File: packages/web-components/src/book-valuation-widget/client/core/__tests__/__snapshots__/index.test.ts.snap [hunk 3]
```diff
@@ -16,9 +16,11 @@ Book_valuation_widget {
       [Function],
       [Function],
       Object {
+        "advanceSearchButton": "css-q9ua3x",
```
Comment (willmcvay) [inline packages/web-components/src/book-valuation-widget/client/core/__tests__/__snapshots__/index.test.ts.snap:19]: Should be featureButton as per my comment

File: packages/elements/src/styles/components/form.scss (hunk 2 above)
Comment (phmngocnghia) [reply packages/elements/src/styles/components/form.scss]: I accidentally changed elements styles. Proceed to revert

File: packages/web-components/src/book-valuation-widget/client/core/__tests__/__snapshots__/index.test.ts.snap (hunk 3 above)
Comment (phmngocnghia) [reply packages/web-components/src/book-valuation-widget/client/core/__tests__/__snapshots__/index.test.ts.snap]: updated

**Pull request merged**